modules/analytics/components/Analytics.tsx

```diff
@@ -85,6 +86,27 @@ function Analytics() {
         }
     });
 
+    if (newsroom?.ga_tracking_id) {
+        return (
+            <Head>
+                <script
+                    async
+                    src={`https://www.googletagmanager.com/gtag/js?id=${newsroom.ga_tracking_id}`}
+                />
```

Comment:
There is a [Script](https://nextjs.org/docs/basic-features/script) component in Next that can be used here instead :)